web/src/store/index.ts

```diff
@@ -27,6 +27,8 @@ export interface State {
     datasetEmbeddings: DatasetEmbedding[];
     datasetTileMetadata: { [key: string]: TileMetadata };
     rootDatasetFrames: TiffFrame[];
+    selectionMode: boolean;
+    subimageSelection: number[] | null;
```

Comment:
I think these should be reset to false/null when the root dataset changes. If I select a region and then change the root dataset from the drop down, I still see the button prompting me to save, and I think this should effectively cancel the selection.

I see similar behavior with the following workflow: Investigate an investigation -> Click the subimage selection button -> select a region -> go back to investigation list view -> investigate an investigation. The button to save the subimage is still there, but the selection is not, and I may be in an entirely different investigation.

Reply:
Changed in [3624b38](https://github.com/atlascope/atlascope/pull/127/commits/3624b38bb27b2e690b1bc7f082c81266042f1462)